pkgs/by-name/pi/pilipala/package.nix

```diff
@@ -0,0 +1,126 @@
+{
+  autoPatchelfHook,
+  lib,
+  fetchFromGitHub,
+  flutter319,
+  mpv,
+  libass,
+  ffmpeg,
+  libplacebo,
+  libunwind,
+  shaderc,
+  vulkan-loader,
+  lcms,
+  libdovi,
+  libdvdnav,
+  libdvdread,
+  mujs,
+  libbluray,
+  lua,
+  rubberband,
+  libuchardet,
+  zimg,
+  alsa-lib,
+  openal,
+  pipewire,
+  libpulseaudio,
+  libcaca,
+  libdrm,
+  mesa,
+  libXScrnSaver,
+  nv-codec-headers-11,
+  libXpresent,
+  libva,
+  libvdpau,
+  pkg-config,
+  makeDesktopItem,
+  wrapGAppsHook3,
+  copyDesktopItems,
+}:
+flutter319.buildFlutterApplication rec {
+  pname = "pilipala";
+  version = "1.0.24-unstable-2024-11-13";
```

Comment:
Obviously when someone else is reading this, the question would arise: why not using a released version?